node/cmd/ccq/http.go

```diff
@@ -112,7 +112,8 @@ func (s *httpServer) handleQuery(w http.ResponseWriter, r *http.Request) {
 		Signature:    signature,
 	}
 
-	if status, err := validateRequest(s.logger, s.env, s.permissions, s.signerKey, apiKey, signedQueryRequest); err != nil {
+	status, queryReq, err := validateRequest(s.logger, s.env, s.permissions, s.signerKey, apiKey, signedQueryRequest)
+	if err != nil {
```

Comment:
Is it necessary to check if queryReq is nil?

Reply:
The function always returns queryReq when the error is nil, so I don't think it's necessary to check the pointer for nil.